plugins/module_utils/redfish_utils.py

```diff
@@ -63,11 +69,23 @@ def get_request(self, uri):
         req_headers = dict(GET_HEADERS)
         username, password, basic_auth = self._auth_params(req_headers)
         try:
-            resp = open_url(uri, method="GET", headers=req_headers,
-                            url_username=username, url_password=password,
-                            force_basic_auth=basic_auth, validate_certs=False,
+            #To support HPE operations
+            if os.path.isfile(home + "/sessionfile.txt"):
+            # if "Hpe" in self.creds.keys():
+                f = open(home + "/sessionfile.txt")
+                session_id = f.readline()
+                resp = open_url(uri, method="GET", headers=GET_HEADERS, sessionid= session_id,
```

Comment:
There seems to be a lot of repeated code here (since you do a very similar modifications in multiple places). Would it be possible to refactor this so that the `open_url` call is done from a common helper function with less repetition?

Reply:
This is just one more call added to each operation as an extra argument (sessionid) is added in it. If at all really needed, changes will be made as enhancements later on. Could you please approve the PR for the moment as the we have developed playbooks with session operations and they depend on this function.